components/Layout/index.jsx

```diff
@@ -30,6 +32,20 @@ const NavigationBar = ({ children }) => {
               </Button>
             </ConfigProvider>
           )}
+          <Popover
+            // defaultOpen
+            title="Notifications"
+            placement="top"
+            // placement="topRight"
+            trigger={['click']}
+            content={Notifications}
+            overlayStyle={{ width: '500px' }}
+          >
+            <Badge dot>
```

Comment:
If we don't have a way to know that there are unread notifications, we shouldn't show the dot.